xarray/tests/test_conventions.py

```diff
@@ -442,3 +443,22 @@ def test_decode_cf_variable_with_array_units(self) -> None:
         v = Variable(["t"], [1, 2, 3], {"units": np.array(["foobar"], dtype=object)})
         v_decoded = conventions.decode_cf_variable("test2", v)
         assert_identical(v, v_decoded)
+
+
+def test_decode_cf_variable_timedelta64():
+    variable = Variable(["time"], pd.timedelta_range("1D", periods=2))
+    decoded = conventions.decode_cf_variable("time", variable)
+    assert decoded.encoding == {}
```

Comment:
```suggestion
    assert decoded.encoding == {}
    assert variable is decoded
```
(not entirely sure if that actually works)

Reply:
Thanks!  It turns out it doesn't -- the `as_variable` call gets in the way -- but I agree with the spirit of the suggestion.  I added `assert_identical` checks to each of these tests.